### ISSUE TITLE
Possible to override buttons?

### ISSUE THREAD

**Issue by MrChriZ** [Crabar/ember-social-share#22]
Is it possible to override the default buttons that are used?

**Comment by Crabar** [comment]
For sure you can change styles. Each button has its own css class. But I don't know about changing templates from outside. Probably it is not possible now.

**Comment by MrChriZ** [comment]
Thanks.  I've just tried overriding the HBS templatefile that didn't work as the classnames seem to be pulled in from the JS.  I'll keep playing. :)

**Comment by MrChriZ** [comment]
So this caused me a bit of a headache but I got there eventually.  Thought I'd just write back here before closing the comment.
Using this video: https://www.youtube.com/watch?v=LIsA0LRMFag
I created a replacement template file which I put in my template.  This got me so far but I had problems that the classes 'share-button' and 'fb-share-button' in the case of the FB button were still pulled in.
I tried setting classNames to '' or '[]' at the point of using the template but that didn't work.

I then tried to prevent ember-social-shares CSS from being imported into my project so that  these classes didn't effect my buttons.  This proved difficult to do. I tried to use Ember-Cli-Funnel to exclude the CSS... but that didn't work.  I've created an issue on the Ember-Cli-Funnel page but as yet it's still an unknown as to why.
Finally I revisited the youtube video above which indicates at about 5:10 how to effectively sub-class the javascript.  So I did this and then used didReceiveAttrs to clear the classNames like so:

```
import FBButton from 'ember-social-share/components/fb-share-button';

export default FBButton.extend({    
    didReceiveAttrs: function() {
        this.set('classNames',[]);
    }
});

```

And low and behold... It worked!  
I guess it would be easier if the addon had a includeStyles flag or something.

Anyhow will close this off for now.

Cheers,
Chris.
